yarn.lock

```diff
@@ -1265,6 +1265,179 @@
   resolved "https://registry.yarnpkg.com/@emotion/weak-memoize/-/weak-memoize-0.1.3.tgz#b700d97385fa91affed60c71dfd51c67e9dad762"
   integrity sha512-QsYGKdhhuDFNq7bjm2r44y0mp5xW3uO3csuTPDWZc0OIiMQv+AIY5Cqwd4mJiC5N8estVl7qlvOx1hbtOuUWbw==
 
+"@expo/bunyan@3.0.2":
```

Comment:
Looks good.

The only thing that worries me a bit is that 1600 new lines in `yarn.lock`. 
But it seems to be expo related, so it is probably fine.

When I try to run `yarn install` I see few `babel` related changes in lock file.
<img width="619" alt="screenshot 2019-03-07 at 10 54 48" src="https://user-images.githubusercontent.com/2660330/53948349-e53a8d00-40c7-11e9-9fb8-ece6a5ba8ef6.png">

Reply:
Refreshed the `yarn.lock` file.

Reply:
Thanks, now it's ok.